db/pf-schema-X.Y.Z.sql

```diff
@@ -114,6 +114,8 @@ CREATE TABLE `node_category` (
   `fingerbank_dynamic_access_list` varchar(255) default NULL,
   `acls` TEXT NOT NULL,
   `inherit_vlan` varchar(50) default NULL,
+  `inherit_role` varchar(50) default NULL,
+  `inherit_web_auth` varchar(50) default NULL,
```

Comment:
That's me being picky but since in switches its called Url, we should call this inherit_web_auth_url, otherwise its name doesn't refer to the URL field and one could think that it refers to something else like the external portal enforcement setting on the switch